src/twisted/conch/scripts/cftp.py

```diff
@@ -181,40 +190,42 @@ def _dispatchCommand(self, line):
         if f is not None:
             return defer.maybeDeferred(f, rest)
         else:
-            self._ebCommand(failure.Failure(NotImplementedError(
-                "No command called `%s'" % command)))
+            errMsg = "No command called `%s'" % (command)
+            self._ebCommand(failure.Failure(NotImplementedError(errMsg)))
```

Comment:
I separated the variables out to facilitate debugging as I was modifying the code.